src/api/db/structure.sql

```diff
@@ -971,21 +971,30 @@ CREATE TABLE `reviews` (
   `created_at` datetime NOT NULL,
   `updated_at` datetime NOT NULL,
   `review_id` int(11) DEFAULT NULL,
-  `reviewable_type` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-  `reviewable_id` int(11) DEFAULT NULL,
-  PRIMARY KEY (`id`),
-  KEY `index_reviews_on_creator` (`creator`) USING BTREE,
-  KEY `index_reviews_on_reviewer` (`reviewer`) USING BTREE,
-  KEY `index_reviews_on_by_user` (`by_user`) USING BTREE,
-  KEY `index_reviews_on_by_group` (`by_group`) USING BTREE,
-  KEY `index_reviews_on_by_project` (`by_project`) USING BTREE,
-  KEY `index_reviews_on_by_package_and_by_project` (`by_package`,`by_project`) USING BTREE,
-  KEY `bs_request_id` (`bs_request_id`) USING BTREE,
-  KEY `index_reviews_on_state_and_by_project` (`state`,`by_project`) USING BTREE,
-  KEY `index_reviews_on_state_and_by_user` (`state`,`by_user`) USING BTREE,
```

Comment:
Seeing the BTREE indexing disappear makes me wonder if have to set this manually in the raw sql queries we run in the migration.